src/Game/GamePage.js

```diff
@@ -3,27 +3,37 @@ import "./Game.css";
 import AttackButton from "./AttackButton";
 import { supabase } from "../supabase_client";
 import AttackTargetSelect from "./AttackTargetSelect";
+import PlayerLocations from "./PlayerLocations";
 
 const GamePage = (props) => {
   const { roomCode, playerName } = props;
   const [totalPlayerData, setTotalPlayerData] = useState([]);
   const [attackTarget, setAttackTarget] = useState("Select a target");
   const [currentTurnPlayerIndex, setCurrentTurnPlayerIndex] = useState(0);
+  const [currentPlayerLocationID, setCurrentPlayerLocationID] = useState(0);
 
   useEffect(() => {
     const fetchHitpoints = async () => {
       const { data } = await supabase
         .from("Players")
-        .select("name, Hitpoints")
+        .select("name, Hitpoints, LocationID")
```

Comment:
* would add in columns that we do not currently need, but it would not be a big issue since we can just call the columns we need from the data returned. But since we only need these three columns I specified what to pull from the database